content/issues/1/_index.md

```diff
@@ -1,10 +1,11 @@
 ---
-layout: issue
 type: issue
+layout: single
 title: Issue One
-number: One
+number: 1
```

Comment:
I meant to look where this was used and forgot. I think we should store as a numeral so we can sort if necessary (although I suppose we could always sort on publish date). I expect there should be a way to generate the text version from the numeral in Go templates; willing to look into that if it's helpful/important.